### DISCUSSION TITLE
Tenacity is back!

### DISCUSSION THREAD

**Discussion by generic-pers0n** [tenacityteam/tenacity-legacy#740] [Announcements]
Today marks another point in the project history of Tenacity: development has officially restarted and is now on Codeberg!

It all started when @FrostKnight made a comment [here](https://github.com/tenacityteam/saucedacity/issues/21#issuecomment-1180822875). They noticed that Saucedacity was still being maintained and had activity. Long story short, discussions between I and the now previous maintainers of Tenacity had occurred, merges happened, and now we're restarting development with a new codebase (Saucedacity's) on Codeberg, which you can now visit here at https://codeberg.org/tenacityteam/tenacity!

This day marks so many changes with Tenacity. Not only have new people come to take their shot at maintaining the project but also the move to Codeberg and a change in features too. There are so many new things regarding Tenacity now, not just feature wise but also project wise! Of course, there are so many to list here, but the most notable ones include using Saucedacity's codebase (which contains features from Audacity 3.1) and moving to Codeberg. The most important thing, however, is focusing on our upcoming 1.3 release, which will be Tenacity's first release. We hope to bring a couple of new things to the table, and we hope to follow that up with the succeeding 1.4 release and beyond.

Tenacity 1.3. is going to be different from all other Audacity forks up to this point. It will include features from Audacity 3.1, namely with clip titles and clip trimming. Under the hood, we will see many more custom libraries, which come from upstream's refactoring efforts. We'll also make our own refactoring effort too, such as moving allocations out of real-time code (namely in AudioIO) and overall general cleanups with the codebase. We hope that our efforts will pay off long-term, but now is the time to start, so it's time to get started!

We are also welcoming someone new to our team: @AnErrupTion! They were the maintainer of Audacium, and recently, they announced that Audacium would be merged into Tenacity. They already made their first PR [here](https://codeberg.org/tenacityteam/tenacity/pulls/82) regarding minor refactoring in almost every part of the codebase. Meanwhile, we will be merging Audacium's themes into Tenacity, so we will definitely be gaining a lot more themes than the extra 'Saucedacity' theme we inherited from the Saucedacity-Tenacity merges for sure! Finally, we'd like to welcome all Audacium users to Tenacity! Tenacity 1.3 will have the same features as Audacium but will also have usability improvements over Audacium and extra features.

Finally, we have to look back to the first couple of months of all these first projects. Saucedacity, Audacium, and Tenacity were fresh off the block among others like AudMonkey. As things progressed, I've seen articles where it was said that there wasn't a "clear alternative" to Audacity based on its initial controversies, especially with the failure to initially unite all of the various forks. Now, looking back in comparison, while those statements had some valid points, things have definitely changed. (One could go as far as to say those statements "aged like milk" even, but I'm not going to go that far). Regardless, knowing that we all had common goals and values, it was a good idea for us to unite under a single project, combining everyone's efforts towards making the same thing.

Overall, today is an exciting day, but we have work to do. It's great that 3 Audacity forks are literally merging together and that Tenacity development is restarting, but as Linus Torvalds once said, "Talk is cheap. Show me the code." Essentially, although we are having exciting news, the news will mean nothing if there's no work that has been done. This work will take time, though, so we are not very eager to let news outlets know. If they cover these events, however, then we'll take it for granted; people will take note of this news, and there's not much we can do to stop that. (If you do ask us first before publishing an article, however, we will greatly appreciate, respect, and applaud you for reaching out first). But the main focus is clear: **although there is some great news to focus on, we have to supplement that by working on the code itself**. However, we cannot neglect to look at the work that is also being dun, such as @AnErrupTion's code cleanup PR. **TD;DR, working towards a new release is our number one priority right now, and work is being done right now as this is being written**.

Before I finish this announcement, however, I have some common points for everyone:

* If you are a Saucedacity or Audacium user, we recommend you move to Tenacity once there's a release out. Both are not going to be maintained any further as they have either been merged into Tenacity or are going to be merged into Tenacity.

* If you are a **Tenacity** packager that has packaged Tenacity 0.1.0 nightly, **we highly recommend that you update your packages to include Tenacity 1.3 nightly**. You will also need to update the package's dependencies appropriately:

  * **wxWidgets**: >= 3.1.6 is now supported, so you may drop any and all patches that used to be required in order to get Tenacity to build with these versions. **However, <= 3.1.3 is no longer supported, so you will need to use at >= 3.1.3 in order to get Tenacity to build**.

* If you were a **Saucedacity or Audacium** packager that packaged a version of Saucedacity or Audacium, whether nightly or stable, **we recommend you make a new package for Tenacity if one doesn't exist**. Chances are, however, that a Tenacity package exists on the same platform you are packaging Saucedacity or Audacium, so another Tenacity package won't be needed. If not, however, please the previous point above.

So let everyone know (**provided our guidelines**; we only want a little publicity right now) that Tenacity is back! We are excited to finally be back and are working on bringing a new 1.3 release that will contain features from Saucedacity and Audacium in addition to a new team consisting of Tenacity contributors and the lead maintainers of Saucedacity (which was me! :D) and Audacium. We welcome you back to Tenacity and hope to see you around [on Codeberg!](https://codeberg.org/tenacityteam/tenacity)

**Comment by RiedleroD** [comment]
just letting you know that you misspelt https://codeberg.org/tenacityteam/tenacity and messed up the link formatting

**Reply by generic-pers0n** [reply]
Whoops! Thanks for catching that!

update: fixed!

**Comment by FrostKnight** [comment]
Glad I could be of service, audacity has become... very audacious...

:P
